tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -92,33 +92,62 @@ public function createLinkShare(string $user, TableNode $body): ResponseInterfac
 	}
 
 	/**
-	 * @When /^user "([^"]*)" gets permissions list for (folder|file) "([^"]*)" of the space "([^"]*)" using the Graph API$/
-	 *
 	 * @param string $user
 	 * @param string $fileOrFolder   (file|folder)
 	 * @param string $resource
 	 * @param string $space
 	 *
-	 * @return void
+	 * @return ResponseInterface
 	 * @throws Exception
 	 */
-	public function theUserPermissionsListOfResource(string $user, string $fileOrFolder, string $resource, string $space):void {
+	public function theUserGetsPermissionsListOfResourceOrSpaceUsingTheGraphApi(string $user, string $fileOrFolder, string $resource, string $space):ResponseInterface {
 		$spaceId = ($this->spacesContext->getSpaceByName($user, $space))["id"];
 
 		if ($fileOrFolder === 'folder') {
 			$itemId = $this->spacesContext->getResourceId($user, $space, $resource);
 		} else {
 			$itemId = $this->spacesContext->getFileId($user, $space, $resource);
 		}
+
+		return GraphHelper::getPermissionsList(
+			$this->featureContext->getBaseUrl(),
+			$this->featureContext->getStepLineRef(),
+			$user,
+			$this->featureContext->getPasswordForUser($user),
+			$spaceId,
+			$itemId
+		);
+	}
+
+	/**
+	 * @When /^user "([^"]*)" gets permissions list for (folder|file) "([^"]*)" of the space "([^"]*)" using the Graph API$/
+	 *
+	 * @param string $user
+	 * @param string $fileOrFolder   (file|folder)
+	 * @param string $resource
+	 * @param string $space
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function theUserGetsPermissionsListOfResource(string $user, string $fileOrFolder, string $resource, string $space):void {
```

Comment:
```suggestion
	public function userGetsPermissionsListForResourceOfTheSpaceUsingTheGraphiAPI(string $user, string $fileOrFolder, string $resource, string $space):void {
```
Using the function name generated by behat will be good and unique for a function under the steps.